network.json

```diff
@@ -19,7 +19,7 @@
 				},
 				"nwType": {
 					"type": "string",
-					"format": "^(infra|data)$",
+					"format": "^infra$|^data$|^$",
```

Comment:
Its easy to specify 'default' value for a field. You can see https://github.com/contiv/contivmodel/blob/master/rule.json#L39 for an example

btw, Here is some documentation on son schema attributes
https://github.com/contiv/contivmodel/blob/master/rule.json#L39